extensions/stackdriver/log/logger.cc

```diff
@@ -341,6 +364,17 @@ void Logger::fillAndFlushLogEntry(
       (*label_map)["upstream_transport_failure_reason"] =
           request_info.upstream_transport_failure_reason;
     }
+    if (!request_info.response_code_details.empty() ||
+        !request_info.connection_termination_details.empty()) {
+      const std::string response_details =
+          !request_info.response_code_details.empty()
+              ? request_info.response_code_details
```

Comment:
I think you could re-use the same field for both.

Reply:
Thanks.. Fixed